docs/integrations/development.md

```diff
@@ -1143,14 +1143,45 @@ For convenience, steps can allow be provided as a comma delimited list.
 
 ex: `j1-integration collect --step step-fetch-users,step-fetch-groups`
 
-###### `--ignore-step-dependencies`
+###### Option `--use-dependencies-cache` or `-C`
 
-If you only want to run a single step or an explicit list of steps without
-invoking the dependencies of those steps, you can do so via the
-`--ignore-step-dependencies` flag. This is useful for speeding up testing by
-utilizing the data that has already been collected and stored on disk.
+Allows preceding steps to skip execution and instead load previously captured results from disk.
+The intent of this is to increase development speed for new integrations.
+When no filepath is specified, an attempt to create a cache is made by
+copying the contents of `./.j1-integrations/graph` directory to `./.j1-cache`. 
+The structure of the cache follows a similar format as the .j1-integration data storage, as described [here](#data-collection).
 
-#### `j1-integration sync`
+###### And example of the expected cache structure
+```
+.j1-cache/
+   /step-fetch-accounts
+      /entities/
+         11fa25fb-dfbf-43b8-a6e1-017ad369fe98.json
+   /step-fetch-users
+      /entities
+         9cb7bee4-c037-4041-83b7-d532488f26a3.json
+         96992893-898d-4cda-8129-4695b0323642.json
+      /relationships
+         8fcc6865-817d-4952-ac53-8248b357b5d8.json
+```
+
+ex: `j1-integration collect --step fetch-users --use-dependencies-cache` - Builds & uses cache from .j1-integration  
+ex: `j1-integration collect --step fetch-users --use-dependencies-cache ./` - Uses .j1-cache found in the root of the project  
+ex: `j1-integration collect --step fetch-users --use-dependencies-cache ./path-to-cache` - Uses .j1-cache found in the path specified 
+
+A common use pattern: 
+1. Execute collection command _without_ the `--use-dependencies-cache` option to gather data in .j1-integration
+2. Execute collection command with `--step` and `--use-dependencies-cache` option _without_ specifying a filepath.
+ This will cause the .j1-integration data to populate the .j1-cache.
+3. Execute collection command with `--step` and `--use-dependencies-cache` option specifying a 
+filepath to the previously created .j1-cache (most commonly `./`).
```

Comment:
I have the same question as Nick here. If this is true, I suppose running step 2 is essentially "refreshing the cache". I was going to suggest we add an option to clear and refresh the cache but sounds like maybe this would handle the refreshing part anyway. I'm just not sure how intuitive that is vs a separate flag needed to refresh the cache in here and otherwise it will default to using the cache that already lives in the directory. Thoughts?

Reply:
@ndowmon Yes. if you repeat step 2 it will copy over the .j1-integration data over and over and only execute the fetch-users step. If the .j1-integration data goes sideways, you'll end up with a cache that is busted. Hence the suggestion to specify the filepath - then it'll stop trying to create the cache.

@ceelias It isn't clear. I thought about a command as well but that seemed too much. I imagine caches being self-maintaining. Not sure if that's the correct approach. Right now the cache creation destroys the previous cache. We could do more of an update, maybe?.... That might be tricky to get right.

Reply:
@ndowmon @ceelias 
Crazy idea: what if we turn on caching by default whenever the `--step` flag is used? We could introduce a simple `--no-cache` flag and `-cache-path` flag. I think this would eliminate the confusion around how caching works currently.

The --step flag is already used to decrease developer wait time... Why not make it ever greater with the cache?

Reply:
🤔 hm, that is an interesting idea... although I do think you may be stuck in cacheland 😉 

My concern with defaulting to `--no-cache` is that differences in upstream dependencies _may_ (or probably _should_) cause the chosen `--step` to behave differently. I still think it's best to refresh the dependent steps before running the new step.